config/crd/bases/argoproj.io_application.yaml

```diff
@@ -0,0 +1,1709 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Strange that this is here.
Does this mean that when you deploy this it also deploys the Application CRD?

Reply:
Moved into `hack`